website/docs/developer/deployment-guide/deploying-cumulus-dashboard-from-mac.md

```diff
@@ -0,0 +1,127 @@
+---
+id: deploying-from-mac
+title: Deploy Cumulus dashboard from Mac
+description: A concrete set of instructions on deploying Cumulus dashboard from Mac to create a test environment for ORCA. Also includes instructions for deploying the dashboard with launchpad authentication.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The goal of this page is to provide a set of additional instructions that will be useful to deploy the dashboard locally from Mac for ORCA project. It is assumed that Cumulus and ORCA are already deployed before following these instructions. Follow instructions on the following link on how to deploy Cumulus and ORCA:
+
+ - https://nasa.github.io/cumulus/docs/deployment/deployment-readme
+ - https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-with-cumulus
+
+## Setting up tunneling to locally run the dashboard
+
+There are several steps involved to set up tunneling to use the dashboard locally.
+
+- Create a pac file named `aws-proxy.pac` under your `user/.ssh` directory and add the following function to that file:
+
+```bash
+function FindProxyForURL (url, host) {
+ // Redirect for Cumulus-ORCA Dashboard
+if (dnsDomainIs(host, '<YOUR_API_ENDPOINT_URL>')) {
+ return 'SOCKS5 localhost:1337';
+}
+// by default use no proxy
+return 'DIRECT';
+}
+```
+
+  Replace `<YOUR_API_ENDPOINT_URL>` with your Cumulus API endpoint found from terraform output named as `archive_api_uri` under module `cumulus-tf`.
+
+1. Firefox browser is used to view the dashboard. On Firefox browser, go to `Setting`-> `Network Settings`. Under `Automatic proxy configuration URL`, add the path to your `aws-proxy.pac` file in this format `file:///Users/<username>/.ssh/aws-proxy.pac`. Replace `<username>` with yours.
+2. Connect to NASA VPN, then run the following command from your bash terminal:
+  ```bash
+  aws ssm start-session --target i-0edf895d1b2f8447d --document-name AWS-StartPortForwardingSession --parameters portNumber=22,localPortNumber=6868
+  ```
+
+  Replace instance-id `i-0edf895d1b2f8447d` with your Cumulus deployed EC2 instance ID. You can get this from EC2 dashboard in AWS and should be named `<prefix>-Cumulus-ECSCluster`.
+3. On another bash terminal, run 
+    ```bash
+    ssh -vvv -p 6868 -N -D 1337 -i ~/.ssh/<YOUR_AWS_PRIVATE_KEY.pem> -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" ec2-user@127.0.0.1
+    ```
+
+  Replace `<YOUR_AWS_PRIVATE_KEY.pem>` with yours. The pem file should have been created while deploying the `cumulus-tf` module via terraform. If not, go to your EC2 dashboard from AWS console and click `Key Pair` field. Then create a new EC2 key pair with your prefix name and download the `.pem` file to your `user/.ssh` directory.
```

Comment:
These instruct will not work as is. You will need to redeploy with the newly created key pair. It seems like this should maybe just be pointed to in a Mac install instruction that is similar to the windows one?

Reply:
or I can just add `Then redeploy your cumulus-tf module via terraform.`

currently there is no webpage for `Mac install instruction`. The only thing we have is the one from cumulus https://nasa.github.io/cumulus/docs/deployment/deployment-readme

Reply:
You may want to add some additional instructions that the `terraform.tfvars` `key_name` value must match the new key pair name before you redeploy.

Reply:
added that under `important` section